### ISSUE TITLE
rust 1.6 compile error

### ISSUE THREAD

**Issue by nacika-ins** [programble/scraper#1]
Such an error has occurred.

```bash
rs:31:68: 31:79 error: mismatched types:
 expected `&mut cssparser::parser::Parser<'_, '_>`,
    found `&mut cssparser::parser::Parser<'_, '_>`
(expected struct `cssparser::parser::Parser`,
    found a different struct `cssparser::parser::Parser`) [E0308]
```

**Comment by programble** [comment]
There are new versions of the Servo dependencies that are causing problems. Updating is non-trivial since the interface have changed. Will try to fix soon.

**Comment by programble** [comment]
Fix released in `v0.1.1`. Updating to newer Servo crates will be `v0.2.0`.

**Comment by nacika-ins** [comment]
thanks!
